README.md

```diff
@@ -21,11 +21,13 @@ Basically it disables the routes and updates the admin and frontend templates.
 ## Installation
 
 ```bash
-composer require monsieurbiz/sylius-no-commerce-plugin="1.0.x-dev" --no-scripts
+composer require monsieurbiz/sylius-no-commerce-plugin --no-scripts
 ```
 
-Change your `config/bundles.php` file to add the line for the plugin : 
+<details><summary>For the installation without flex, follow these additional steps</summary>
```

Comment:
Don't forget to explain the `--no-scripts` here, for composer.

Reply:
with the fix for the default configuration, this option is useless